app.rb

```diff
@@ -1,9 +1,24 @@
 require 'sinatra/base'
+require './lib/message.rb'
 
 class Chitter < Sinatra::Base
   get '/test' do
```

Comment:
It's good to remove any extraneous code from your implementation before making a Pull Request - there'll be lots of testing and exploration as you work on something, but it tidies it up to remove anything that is no longer needed for the program to meet its specification.